### ISSUE TITLE
net.box hangs during loading scheme

### ISSUE THREAD

**Issue by rtsisyk** [tarantool/tarantool#1275]
A regression introduced by #1183 feature:

```
1$ ../src/tarantool 
../src/tarantool: version 1.6.8-311-g6c80719
type 'help' for interactive help
tarantool> box.cfg { listen = 3313 }
```

```
2$ ../src/tarantool 
../src/tarantool: version 1.6.8-311-g6c80719
type 'help' for interactive help
tarantool> require('net.box').new('127.0.1', 3313)                                                                                      

<!-- hangs
```

A connection to the same Tarantool works properly.
It seems that net.box needs more tests.

**Comment by Sulverus** [comment]
After little research:
in new net.box we use box.schema.VSPACE_ID, it doesn't works without box.cfg{}.

**Comment by Sulverus** [comment]
merged to 1.6 branch
